lib/VCMIDirs.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include "StdInc.h"
 #include "VCMIDirs.h"
+#include "../lib/json/JsonNode.h"
```

Comment:
but you're already inside `lib`

Reply:
sorry, what do you mean? I'm taking about
```suggestion
#include "json/JsonNode.h"
```